www/pages/_app.tsx

```diff
@@ -1,6 +1,8 @@
 import { AppProps } from "next/app";
 import React, { useEffect, useState } from "react";
 
+import "../styles/global.css";
```

Comment:
😱 Plain old CSS without sixteen hundred layers of magic? Is that still legal in 2022?

Reply:
Only way to reach `body` -.-"

Reply:
Could be at least SASS *grumble grumble*